### DISCUSSION TITLE
articles/elon-musk

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#183] [Comments]
# articles/elon-musk

约瑟夫·坎贝尔在《千面英雄》中总结了英雄神话的模式：在英雄还是普通人的时候，他受到召唤启程冒险，起初抵触，进而与智者相遇得到启蒙，跨越边界，开始初次尝试，到历险阶段历经严峻考验，由内而外得到救赎，胜利回归。好莱坞影片中的英雄故事无不是按照这个模式来演绎，蜘蛛侠、蝙蝠侠、阿凡达……皆是如此。相对于编剧的精心包装，真实世界里的英雄故事则要复杂的多，但追寻他们的成长轨迹，仍能找到“启程—>启蒙—>回归”的循环模式，只是这一模式会在他们的故事中重复多次。

https://hutusi.com/articles/elon-musk

**Comment by 0X-NPC** [comment]
很详细，👍。「而后退出Model 3」应该是「而后推出Model 3」的意思吧？

**Reply by hutusi** [reply]
谢谢，fixed。很仔细👍

**Comment by ethan-jo** [comment]
「对于那些想成为我的人，我不确定是否真的想成为我这样的人。坦率的说，你永远想象不到我对自己的折磨已经到了什么程度。」
马斯克目标感极强，没有强大的精神世界无法驾驭所面对的局面，过程中必然经历极端痛苦，如果这些经历降临普通人实在难言是天堂还是地狱。

**Reply by hutusi** [reply]
是呀，需要强大心理~~
